rejection-handler/src/main/java/org/motechproject/nms/rejectionhandler/service/VillageRejectionService.java

```diff
@@ -0,0 +1,11 @@
+package org.motechproject.nms.rejectionhandler.service;
+
+import org.motechproject.nms.rejectionhandler.domain.VillageImportRejection;
+import org.springframework.stereotype.Service;
+
+public interface VillageRejectionService {
+
+    Long saveRejectedVillage(VillageImportRejection villageImportRejection);
```

Comment:
Is save and create both inserting? Why two different names?

Reply:
a unique key is implemented for location code + state code, so if we dont recive location code in any data... we have to use create method which is not able to update records, while save works for updating if any such record exists,